src/generate/project.rs

```diff
@@ -376,7 +376,7 @@ disable_stacks_api = false
 # bitcoin_explorer_image_url = "quay.io/hirosystems/bitcoin-explorer:devnet"
 # postgres_image_url = "postgres:alpine"
 # enable_hyperchain_node = true
-# hyperchain_image_url = "hirosystems/hyperchains:103-merge-stretch"
+# hyperchain_node_image_url = "hirosystems/hyperchains:103-merge-stretch"
```

Comment:
Should this match the default? (`hirosystems/hyperchains:0.0.2-stretch`)

Reply:
Addressed with latest